packages/manager/apps/container/src/container/nav-reshuffle/index.tsx

```diff
@@ -26,7 +20,7 @@ function NavReshuffleContainer(): JSX.Element {
   const iframeRef = useRef(null);
   const [iframe, setIframe] = useState(null);
   const [showOverlay, setShowOverlay] = useState(false);
-  const { shell } = useContext(ApplicationContext);
+  const shell = useShell();
```

Comment:
```suggestion
  const { shell } = useApplication();
```
Shouldn't we use the `useApplication` hook (since it basically does the same thing) ?

Reply:
Yes, I very much agree and thought about that. This would require to rewrite some of the tests though, because they explicitly mock the `useShell` hook.

Reply:
As seen together, let's tackle this in your other ticket